### PR TITLE
chore: Updates next 13 branch with main branch - SFS-1207

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

This PR aims to update the Next 13 feature branch with the changes that come from the main branch

JIRA task: https://vtex-dev.atlassian.net/browse/SFS-1207

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -47,6 +47,8 @@ storybook-static/
 node_modules/
 jspm_packages/
 
+!packages/cli/src/__mocks__/**/node_modules
+
```
Comment (emersonlaurentino) [inline .gitignore:51]: **question (non-blocking):** why we need to add node_modules from cli?

Comment (eduardoformiga) [reply .gitignore:51]: I think this is related to the faststore platform initiative. Not introduced in this PR.
Would you happen to know @gvc, @vaporwavie?

Comment (vaporwavie) [reply .gitignore:51]: This was implemented by @gvc but I support what you mentioned @eduardoformiga. It was introduced on this commit that refers adding `core` for monorepos: https://github.com/vtex/faststore/commit/e6e3ab7710ce2418afb07127ceab3c1bb8e2bcfc.

**Review by hellofanny — APPROVED** [review]
Looks fine to me 🚀

**Pull request merged**